package.json

```diff
@@ -84,14 +84,15 @@
     },
     "scripts": {
         "clean": "rimraf out ./lsp/server/out ./lsp/client/out",
-        "compile": "rimraf ./lsp/server/tsconfig.tsbuildinfo ./lsp/client/tsconfig.tsbuildinfo && tsc -b ./",
+        "compile": "rimraf ./lsp/server/tsconfig.tsbuildinfo ./lsp/client/tsconfig.tsbuildinfo && npm run copy-resources && tsc -b ./",
+        "copy-resources": "copyfiles -s -u 3 'lsp/server/resources/*' 'lsp/server/out/resources/'",
```

Comment:
Using `copyfiles` to copy a resource file to the output folder. `tsc` doesn't do this. And we need a utility like `copyfiles` to work on all platforms(Linux/Mac/Windows).

Reply:
You don't really need to do this on all platforms though, do you? This is in the service of developing the extension, not using it. We're not strictly required to support Windows as a development platform for the extension.

Reply:
We need to do this because when lsp is initialized only at that time the yaml file is read in. So the yaml file needs to be packaged.

Reply:
Yes, but it only needs to be packaged at the time of creating the package. We don't have a requirement for a cross-platform way to create the package, just consume it. You could get away with a `cp` here, given that our package creation will almost certainly happen in a *nix environment.